### PR TITLE
DELETE THIS  BA-4 

### PR DESCRIPTION
**Pull request by ivanovakatya**
In this Pull Request the "+" button that was plan to be in the tappable cell in MyReadingList screen in the beginning is replaced with the default arrow. 
Search screen need change too because  is part of the navigation stack.
Simulator Screen Shot - iPod touch (7th generation) - 2021-12-17 at 20 31 13Simulator Screen Shot - iPod touch (7th generation) - 2021-12-17 at 20 26 40

### CODE REVIEW

File: MyReadingList/MyReadingList/MyReadingListView.swift [hunk 1]
```diff
@@ -10,21 +10,19 @@ import SwiftUI
...
-            Text(NSLocalizedString("ui.myReadingList.title", value: "My Reading List", comment: "title for the MyReadingList screen"))
-                .font(.headline)
-            Spacer()
-            
-            List {
-                HStack{
-                    Button {
-                        //do nothing for now
+        NavigationView {
+            VStack {
```
Comment (tricerintops) [inline MyReadingList/MyReadingList/MyReadingListView.swift:14]: Is this VStack necessary?

File: MyReadingList/MyReadingList/MyReadingListView.swift [hunk 2]
```diff
@@ -10,21 +10,19 @@ import SwiftUI
...
+                List {
+                    HStack{
+                        NavigationLink {
```
Comment (tricerintops) [inline MyReadingList/MyReadingList/MyReadingListView.swift:17]: It looks like the code for this NavigationLink is not all properly indented

File: MyReadingList/MyReadingList/MyReadingListView.swift [hunk 3]
```diff
@@ -10,21 +10,19 @@ import SwiftUI
...
+                                    SearchView()
                     } label: {
-                        Image(systemName: "plus.circle")
+                            Image(systemName: "plus.circle")
```
Comment (tricerintops) [inline MyReadingList/MyReadingList/MyReadingListView.swift:20]: You now have "plus" icon in this cell as well as an arrow on the right (run it in the simulator if you are not sure what I mean)

It seems excessive to have 2 indicators that this is a tappable element. I would choose one and get rid of the other. Up to you which one to keep.

**Comment by tricerintops** [comment]
Is this PR missing some changes to the SearchView (related to having a navigation title)?

File: MyReadingList/MyReadingList/MyReadingListView.swift (hunk 1 above)
Comment (ivanovakatya) [reply MyReadingList/MyReadingList/MyReadingListView.swift:14]: In the begin I thought I will need both a VStack and HStack to achieve my task, but ready more about it. I learn that there is no need of VStack. That's why I will remove it in the next commit.

File: MyReadingList/MyReadingList/MyReadingListView.swift (hunk 2 above)
Comment (ivanovakatya) [reply MyReadingList/MyReadingList/MyReadingListView.swift:17]: Thanks a lot for pointing this. I find a shortcut of indenting my code: select the code that need to be indented (have been charged) and press Control + I

File: MyReadingList/MyReadingList/MyReadingListView.swift (hunk 3 above)
Comment (ivanovakatya) [reply MyReadingList/MyReadingList/MyReadingListView.swift:20]: I will keep the default indicators the arrow to describe this is a tappable element and will remove the '+' icon.

File: MyReadingList/MyReadingList/MyReadingListView.swift (hunk 1 above)
Comment (tricerintops) [reply MyReadingList/MyReadingList/MyReadingListView.swift]: Sounds good!

File: MyReadingList/MyReadingList/MyReadingListView.swift (hunk 2 above)
Comment (tricerintops) [reply MyReadingList/MyReadingList/MyReadingListView.swift]: Excellent find!

File: MyReadingList/MyReadingList/MyReadingListView.swift (hunk 3 above)
Comment (tricerintops) [reply MyReadingList/MyReadingList/MyReadingListView.swift]: Sounds good

File: MyReadingList/MyReadingList/MyReadingListView.swift [hunk 4]
```diff
@@ -10,21 +10,16 @@ import SwiftUI
...
-            Text(NSLocalizedString("ui.myReadingList.title", value: "My Reading List", comment: "title for the MyReadingList screen"))
-                .font(.headline)
-            Spacer()
-            
+        NavigationView {
             List {
-                HStack{
-                    Button {
-                        //do nothing for now
+                HStack {
```
Comment (tricerintops) [inline MyReadingList/MyReadingList/MyReadingListView.swift:15]: What is the purpose of this HStack here? (i.e. I don't think it's needed)

File: MyReadingList/MyReadingList/MyReadingListView.swift [hunk 5]
```diff
@@ -10,21 +10,16 @@ import SwiftUI
...
+                    NavigationLink {
+                        SearchView()
                     } label: {
-                        Image(systemName: "plus.circle")
+                        Text(NSLocalizedString("ui.myReadingList.addABook.button", value: "Add a book", comment: "Add a Book button in MyReadingList screen"))
```
Comment (tricerintops) [inline MyReadingList/MyReadingList/MyReadingListView.swift:19]: Personally, I think the code will be easier to read if you put each parameter in the localized string on a newline. Such as :
```
Text(NSLocalizedString("ui.myReadingList.addABook.button", 
value: "Add a book", 
comment: "Add a Book button in MyReadingList screen")
```
(let the editor take care of the proper indentation)

But that is entirely up to you. If you think it's easy to read as is, then leave it as is.

**Pull request closed without merging**